adminSDK/reseller/index.js

```diff
@@ -116,26 +99,20 @@ function storeToken(token) {
  * @param {google.auth.OAuth2} auth An authorized OAuth2 client.
  */
 function listSubscriptions(auth) {
-  const service = google.reseller('v1');
+  const service = google.reseller({version: 'v1', auth});
   service.subscriptions.list({
-    auth: auth,
     maxResults: 10,
-  }, function(err, response) {
-    if (err) {
-      console.log('The API returned an error: ' + err);
-      return;
-    }
-    const subscriptions = response.subscriptions;
-    if (subscriptions.length == 0) {
-      console.log('No subscriptions found.');
-    } else {
-      console.log('Subscriptions:');
-      for (let i = 0; i < subscriptions.length; i++) {
-        const subscription = subscriptions[i];
-        console.log('%s (%s, %s)', subscription.customerId, subscription.skuId,
-            subscription.plan.planName);
-      }
-    }
+  }, (err, {data}) => {
+    if (err) return console.error('The API returned an error:', err.message);
+
+    const subscriptions = data.subscriptions;
+    if (subscriptions.length == 0) return console.log('No subscriptions found.');
+
+    console.log('Subscriptions:');
+    subscriptions.forEach((subscription) => {
+      console.log(subscription.customerId,
```

Comment:
Shouldn't the customer ID also be in the template string?

Reply:
That is a poor attempt at preventing this line from going over 100 chars per line, and I don't want to add new lines in the console. I have something better in mind